.travis.yml

```diff
@@ -1,3 +1,5 @@
 language: node_js
 node_js:
-  - '8'
\ No newline at end of file
+  - '8'
+script:
+  - npm run lint
```

Comment:
I think this overrides the npm run test command for travis, It would be better to add it as
{
    ...
    "scripts": {
        "pretest": "eslint --ignore-path .gitignore ."
    }
    ...
}

https://medium.com/the-node-js-collection/why-and-how-to-use-eslint-in-your-project-742d0bc61ed7

Reply:
Ah ok, you pushed in the meantime, that also works yep
Up to you to decide between adding two steps in Travis or a pretest step in package.json

Reply:
I like this idea @denisgranha  I will change it.